modules/push/src/main/WebPush.scala

```diff
@@ -0,0 +1,43 @@
+package lila.push
+
+import play.api.libs.json._
+import play.api.libs.ws.WS
+import play.api.Play.current
+
+import lila.user.User
+
+private final class WebPush(
+    getSubscriptions: User.ID => Fu[List[WebSubscription]],
+    url: String,
+    vapidPublicKey: String
+) {
+
+  def apply(userId: User.ID)(data: => PushApi.Data): Funit =
+    getSubscriptions(userId) flatMap { subscriptions: List[WebSubscription] =>
+      subscriptions.map(send(_, data)).sequenceFu.void
```

Comment:
can we make a single `send` with several `sub`?

Reply:
done (niklasf/lila-push@c4d51ea5882d8a8db5561750588db18a0a23aae1 and c4d51ea5882d8a8db5561750588db18a0a23aae1)